src/evernodeone.c

```diff
@@ -87,10 +110,6 @@ int64_t hook(uint32_t reserved)
             uint8_t host_addr[HOST_ADDR_VAL_SIZE];
             CLEAR_BUF(host_addr, 0, HOST_ADDR_VAL_SIZE); // Initialize buffer wih 0s
 
-            // <host_address(20)><cpu_model_name(40)><cpu_count(2)><cpu_speed(2)><cpu_microsec(4)><ram_mb(4)><disk_mb(4)>
-            uint8_t token_id[TOKEN_ID_VAL_SIZE];
-            CLEAR_BUF(token_id, 0, TOKEN_ID_VAL_SIZE); // Initialize buffer wih 0s
-
             if (state(SBUF(host_addr), SBUF(STP_HOST_ADDR)) != DOESNT_EXIST)
                 rollback(SBUF("Evernode: Host already registered."), 1);
 
```

Comment:
Can't we move "Generate the NFT token id." to `if (has_initiated_transfer == 0)` block